components/view/ProjectsView.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import { Typography, Avatar, Grid, Card, CardContent } from '@mui/material';
+import { ViewButton, ViewSectionTypography } from 'components/create/Styled';
+
+import { IPortfolio } from 'models/data/IPortfolio';
+
+type ProjectsViewProps = {
+  portfolio: IPortfolio;
+};
+
+const ProjectsView = (props: ProjectsViewProps) => {
+  const { portfolio } = props;
+
+  return (
+    <div>
+      <ViewSectionTypography variant="h3">Projects</ViewSectionTypography>
+
+      {portfolio.content.projects?.map((project, index: number) => (
+        <Card
+          sx={{
+            borderRadius: 7,
+            marginTop: 3,
+          }}
+          key={index}
+        >
+          <CardContent>
+            <Stack
+              key={index}
+              spacing={1}
+              sx={{ paddingBottom: 2 }}
+              direction="row"
+            >
+              <Grid sx={{ flexGrow: 1 }} container direction="row">
+                <Stack direction="row">
+                  <Avatar
+                    variant="rounded"
+                    alt={project.title}
+                    sizes="big"
+                    src={project.picture}
+                    sx={{
+                      width: '100%',
+                      height: '100%',
+                      maxWidth: 300,
+                      maxHeight: 300,
+                    }}
+                  />
+                  <Grid item></Grid>
```

Comment:
Is this necessary? You shouldn't use components just for spacing, that's what CSS is for :)

Reply:
fixed